add fallback to "npm ci" in install job

```bash
npm ci || npm install
```

This is for greenkeeper lockfile update.

```bash
npm ci
npm ERR! cipm can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.
npm ERR! 
npm ERR! 
npm ERR! Invalid: lock file's rollup-plugin-node-resolve@3.2.0 does not satisfy rollup-plugin-node-resolve@3.3.0
npm ERR! 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/circleci/.npm/_logs/2018-03-17T19_00_56_092Z-debug.log
Exited with code 1
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -15,7 +15,7 @@ jobs:
       - run: sudo npm install -g npm@latest
 
       # install
-      - run: npm ci
+      - run: npm ci || npm install
 
       # update lockfile
       - run: npx greenkeeper-lockfile-update

